app/templates/views/guidance/features/security.html

```diff
@@ -9,79 +9,231 @@
+  <p class="govuk-body">All code changes must be reviewed by the team before we can deploy them.</p>
+  <p class="govuk-body">We monitor our production environment for unauthorised changes.</p>
+  <p class="govuk-body">We give our users appropriate notice before:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>any planned outages or downtime</li>
+    <li>making significant functional changes</li>
+  </ul>
+  <p class="govuk-body">We announce planned downtime on the <a class="govuk-link govuk-link--no-visited-state" href="https://status.notifications.service.gov.uk">Notify status page</a>.</p>
+
+  <h2 class="heading-medium" id="finding-and-fixing-security-issues">Finding and fixing security issues</h2>
+  <p class="govuk-body">GOV.UK Notify:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>follows <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/collection/developers-collection/principles">secure development principles</a></li>
+    <li>tracks third-party dependencies in our code base</li>
+    <li>monitors our logs for attacks, misuse and malfunctions</li>
+    <li>provides <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.support') }}">24-hour online support</a></li>
+  </ul>
+  <p class="govuk-body">We use <a class="govuk-link govuk-link--no-visited-state" href="https://www.ncsc.gov.uk/information/web-check">Web Check</a> and other services to:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>scan for vulnerabilities</li>
+    <li>prioritise which software patches to test and deploy first</li>
+  </ul>
+  <p class="govuk-body">AWS is responsible for patching our infrastructure:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>firmware</li>
+    <li>hardware</li>
+    <li>operating system (OS) kernel</li>
+  </ul>
+
+  <h2 class="heading-medium" id="security-incidents">Security incidents</h2>
+  <p class="govuk-body">We provide a 24-hour response in case of an incident.</p>
+  <p class="govuk-body">If there is a data loss event, we will contact you directly.</p>
+  <p class="govuk-body">If there is another type of incident, we’ll publish details and updates on our <a class="govuk-link govuk-link--no-visited-state" href="https://status.notifications.service.gov.uk">system status page</a>.</p>
+
+  <h2 class="heading-medium" id="sign-in-and-api-access">Sign in and API access</h2>
+  <h3 class="heading-small">Signing in to Notify</h3>
+  <p class="govuk-body">GOV.UK Notify uses two-factor authentication for sign-in.</p>
+  <p class="govuk-body">Find out more about the <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_sign_in_method') }}">sign-in method</a>.</p>
+  <p class="govuk-body">You must keep to our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.terms_of_use') }}">terms of use for signing in to Notify</a>.</p>
```

Comment:
@idavidmcdonald would this address the points raised in today’s Slack thread?

```suggestion
  <p class="govuk-body">GOV.UK Notify uses two-factor authentication for sign-in.</p>
  <p class="govuk-body">Team members can sign in with a text message code or a link that’s sent in an email.</p>
  <p class="govuk-body">For security, you’ll need to confirm that you still have access to your email address every few months.</p>
  <p class="govuk-body">Find out more about our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_sign_in_method') }}">sign-in methods</a>.</p>
  <p class="govuk-body">You must keep to our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.terms_of_use') }}">terms of use for signing in to Notify</a>.</p>
```

Reply:
We can be more specific and say `3` instead of `few`.

Reply:
Looks great to me